crates/iceberg/Cargo.toml

```diff
@@ -62,6 +62,8 @@ urlencoding = { workspace = true }
 uuid = { workspace = true }
 
 [dev-dependencies]
+iceberg_test_utils = { path = "../test_utils", features = ["tests"] }
+port_scanner = { workspace = true }
```

Comment:
We don't need this, you can refer to the yaml as an example: https://github.com/apache/iceberg-rust/blob/641d05800e97afe732fe6c8317c01d022e449092/crates/catalog/rest/testdata/rest_catalog/docker-compose.yaml#L39